mathics/builtin/__init__.py

```diff
@@ -88,23 +79,6 @@ def contribute(definitions):
             definitions.builtin[op] = Definition(name=op)
 
 
-def get_module_doc(module):
-    doc = module.__doc__
-    if doc is not None:
-        doc = doc.strip()
-    if doc:
-        title = doc.splitlines()[0]
-        text = "\n".join(doc.splitlines()[1:])
-    else:
-        title = module.__name__
-        for prefix in ("mathics.builtin.", "mathics.optional."):
-            if title.startswith(prefix):
-                title = title[len(prefix) :]
-        title = title.capitalize()
-        text = ""
-    return title, text
-
-
```

Comment:
This is going to require a small import change in mathics_django. This has been moved into common_doc.py since this is used in generating documentation or doctests.